router/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "actix-router"
-version = "0.2.4"
+version = "0.2.5"
```

Comment:
Do NOT change the version number, we usually do it when releasing.

Reply:
on this point, in the changelog, use "unreleased" as the section heading